controllers/openstackbaremetalset_controller.go

```diff
@@ -164,6 +164,7 @@ func (r *OpenStackBaremetalSetReconciler) SetupWithManager(mgr ctrl.Manager) err
 	openshiftMachineAPIBareMetalHostsFn := handler.EnqueueRequestsFromMapFunc(func(_ context.Context, o client.Object) []reconcile.Request {
 		result := []reconcile.Request{}
 		label := o.GetLabels()
+		r.Log.Info(fmt.Sprintf("#####BareMetalHost object triggering reconcile %s", o.GetName()))
```

Comment:
Is this leftover debug or did we want this going forward?

Reply:
I can remove it either completely or remove the prefix hashes. We've it in some controllers though as it helps when looking at controller logs. https://github.com/openstack-k8s-operators/openstack-operator/blob/main/controllers/dataplane/openstackdataplanenodeset_controller.go#L701

Reply:
Removed!